adam/adam_processing_service.py

```diff
@@ -190,10 +190,24 @@ def get_result_ephemeris(self, run_number, force_update=False):
         if not base_url.endswith('/'):
             base_url = base_url + '/'
         url = urllib.parse.urljoin(base_url, ephemeris_resource_name)
-        print(url)
         with urllib.request.urlopen(url) as response:
             return response.read().decode('utf-8')
 
+    def get_result_ephemeris(self, run_number, force_update=False):
+        """Get an ephemeris for a particular run in the batch as a Panda DataFrame
```

Comment:
nit: pandas?

Reply:
I had been using the singular this whole time. I've updated this and the issue text accordingly.

Reply:
I believe this is resolved now.